tests/test_closing.py

```diff
@@ -3642,12 +3642,12 @@ def test_close_weight_estimate(node_factory, bitcoind):
     log = l1.daemon.wait_for_log('sendrawtransaction: ')
     tx = re.match('.*sendrawtransaction: ([0-9a-f]*).*', log).group(1)
 
-    # This could actually be a bit shorter: 1 in 256 chance we get
-    # lucky with a sig and it's shorter.  We have 2 sigs, so that's
-    # 1 in 128.  Unlikely to do better than 2 bytes off though!
+    # To match the signer's estimate we use the pessimistic estimate
+    # of 73bytes / signature, which often ends up overestimating the
+    # actual weight by up to 4 bytes.
     signed_weight = int(bitcoind.rpc.decoderawtransaction(tx)['weight'])
     assert signed_weight <= actual_weight
-    assert signed_weight >= actual_weight - 2
+    assert signed_weight >= actual_weight - 4
```

Comment:
This is confusing!  actual_weight should be called "final_estimate" since that's what it is: closingd's weight estimate for the final negotiated tx.

And this change is wrong. We are *always* 4 bytes under, since we actually grind to 71 bytes per signature, so first assert should be:

```
        assert signed_weight <= final_estimate - 4
```

But occasionally we get lucky (and fail CI like we did here), indeed, decoding shows we got a 70 byte signature, so the second should be:

```
        assert signed_weight >= final_estimate - 6
```

Reply:
Ok, now that makes sense, adjusted as per your comments.